packages/genesys-spark-components/src/components/beta/gux-link/gux-link.tsx

```diff
@@ -0,0 +1,39 @@
+import { Component, Element, h, Host, JSX, Prop } from '@stencil/core';
+import { trackComponent } from '@utils/tracking/usage';
+import { getClosestElement } from '@utils/dom/get-closest-element';
+
+@Component({
+  styleUrl: 'gux-link.scss',
+  tag: 'gux-link-beta',
+  shadow: true
+})
+export class GuxLink {
+  @Element()
+  private root: HTMLElement;
+
+  @Prop()
+  size: 'medium' | 'small' = 'medium';
+
+  @Prop()
+  standalone: boolean = false;
```

Comment:
this seems fine to be my only concern is if the add other types of links in the future potentially like external links or something. maybe a type would be better.

Reply:
I think standalone covers external links (by adding the external link icon to it). I thought to do it that way too, but couldn't think of any other types that could possibly exist style-wise. Everything else is controlled by the user through the slotted <a>.